deploy/README.md

```
@@ -0,0 +1,118 @@
+| DNS rebinding protection | Enabled | `HTTP_ALLOWED_HOSTS` validates Host header |
+| Distroless image | Enabled | `gcr.io/distroless/cc-debian12:nonroot` |
+| Read-only root filesystem | Enabled | `read_only: true` in compose |
+| No new privileges | Enabled | `no-new-privileges:true` security opt |
+| HTTPS termination | Enabled | Tailscale serves TLS via Let's Encrypt |
+| Ephemeral node | Enabled | Node removed from tailnet on shutdown |
+
+---
+
+## Directory layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)